js/index.js

```diff
@@ -1,6 +1,79 @@
-import { format } from "date-fns";
+import { format, getDaysInMonth, getMonth, getYear } from "date-fns";
 import * as R from "ramda";
 
-console.log(format(new Date(), "'Today is a' eeee"));
 
-console.log(R.isNil([]));
+
+// exercise getDaysList //
+/* creare un metodo che riceve in input due valori: 
+month, year e ritorna il numero dei giorni di quel mese */
+
+/* var date = new Date();
+var month = getMonth(date);
+var year = getYear(date);
+
+console.log('mese =' + month + ' ' + 'anno = ' + year) */
+
+function getDaysList(month, year) {
```

Comment:
@sciuto-s come mia c è anche questa funzione in questa pr? te la sei portata dietro da una vecchia issue?

Reply:
no, questa mi sembra sia una funzione che aveva scritto stefano sul suo file..